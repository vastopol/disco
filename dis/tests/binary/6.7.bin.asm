AND R0, R0, x0
ADD R0, R0, x1
AND R1, R1, x0
ADD R1, R1, x1B
AND R3, R3, x0
ADD R3, R3, xA
LD R4, x9
LDR R2, R4, x0
ADD R2, R2, R1
BRz x5
ADD R4, R4, x1
ADD R3, R3, x1F
LDR R2, R4, x0
BRp x1FA
AND R0, R0, x0
TRAP x25
ST R0, x100
