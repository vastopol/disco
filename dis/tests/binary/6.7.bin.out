5020
1021
5260
127B
56E0
16EA
2809
6500
1481
0405
1921
16FF
6500
03FA
5020
F025
3100
